pkg/liquidity-mining/contracts/admin/AuthorizerAdaptorEntrypoint.sol

```diff
@@ -29,6 +29,8 @@ import "@balancer-labs/v2-solidity-utils/contracts/openzeppelin/Address.sol";
  * permissions correctly.
  */
 contract AuthorizerAdaptorEntrypoint is IAuthorizerAdaptorEntrypoint {
+    event ActionPerformed(address indexed caller, address indexed target, bytes data);
```

Comment:
I guess indexing the addresses makes sense since both the authorized callers and the targets are not _that_ many.

Optionally we could also include the selector; even if it's contained in the data perhaps it could be useful?

Reply:
I like indexing the selector!

Reply:
It is possible we'll want to look to e.g. `checkpoint` calls in multiple targets

Reply:
Done.

I've modified the test accordingly. I added some random bytes to the calldata in the test so that we can actually verify that the selector is where it's supposed to be, and it's not just a match by chance. If you think it's not too clear please let me know; holding the merge til then.